assets/main.js

```diff
@@ -9,7 +6,71 @@ class Book {
   }
 }
 
-document.addEventListener('DOMContentLoaded', UI.displayBooks());
+function getBooksFromLocal() {
+  let books;
+  if (localStorage.getItem('books') === null) {
+    books = [];
+  } else {
+    books = JSON.parse(localStorage.getItem('books'));
+  }
+
+  return books;
+}
```

Comment:
[REQUIRED] In this file try using ES6 arrow functions, instead of the function keyword. Also, I recommend you reading about logical operators, it'll make your code look a lot cleaner
```
const getBooksFromLocal = () => JSON.parse(localStorage.getItem('books')) || [];
```

![image](https://user-images.githubusercontent.com/73254040/98421610-51b12080-204f-11eb-9fca-ab11ab0d9ef3.png)